code/datums/keybindings/admin.dm

```diff
@@ -0,0 +1,67 @@
+/datum/keybinding/admin
+	category = KB_CATEGORY_ADMIN
+
+/datum/keybinding/admin/can_use(client/C, mob/M)
+	return !isnull(C.holder) && ..()
```

Comment:
Make this a check_rights() check, otherwise mentors will be able to invoke player panel

Reply:
foiled again.